lib/i18n/hygiene/config.rb

```diff
@@ -8,6 +8,7 @@ class Config
       attr_writer :locales
       attr_writer :keys_to_skip
       attr_writer :concurrency
+      attr_reader :all_locales
```

Comment:
I don't think you need to add `all_locals` as an `attr_reader` since you already defined a method on line 25.